### DISCUSSION TITLE
.Next is also getting pushed on GIthHub

### DISCUSSION THREAD

**Discussion by Boby900** [vercel/next.js#58809] [Help]
### Summary

I have added a `/.next/` in my `.gitignore`  file but still, the **Next** folder is getting pushed to GitHub which is consuming lots of data and my time.
I had to delete that manually.
Any help would be highly appreciated!
![Screenshot (226)](https://github.com/vercel/next.js/assets/111265239/54a88340-9dfb-4fc9-9320-23fd7f5d5479)

**Comment by icyJoseph** [comment]
That's kind of weird... when you run `git status`, what do you see?

**Reply by Boby900** [reply]
It's just showing nothing to commit

![Screenshot (229)](https://github.com/vercel/next.js/assets/111265239/1c55bb55-d058-4764-a601-de919bd7f399)

**Comment by rajatdhoot123** [comment]
might be you need to remove from git
```
git rm -rf .next
```
